pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle/ecdhes_aes_aead_composite_test.go

```diff
@@ -176,6 +185,54 @@ func TestEncryptDecryptNegativeTCs(t *testing.T) {
 	}
 }
 
+func TestEncryptDecryptWithSingleRecipient(t *testing.T) {
+	recipientsPrivKeys, recipientsPubKeys := buildRecipientsKeys(t, 1)
+	aeadPrimitive := getAEADPrimitive(t, aead.AES256GCMKeyTemplate())
+
+	mEncHelper := &MockEncHelper{
+		KeySizeValue: 32,
+		AEADValue:    aeadPrimitive,
+		TagSizeValue: subtleaead.AESGCMTagSize,
+		IVSizeValue:  subtleaead.AESGCMIVSize,
+	}
+
+	pt := []byte("secret message")
+	aad := []byte("aad message")
+
+	// test with single recipient public key
+	cEnc := NewECDHESAEADCompositeEncrypt(recipientsPubKeys, commonpb.EcPointFormat_UNCOMPRESSED.String(),
+		mEncHelper, ecdhespb.KeyType_EC)
+
+	// Encrypt should fail with aad not base64URL encoded
+	_, err := cEnc.Encrypt(pt, aad)
+	require.Error(t, err)
```

Comment:
Should you be checking for certain error type/message?

Reply:
added